go.mod

```diff
@@ -112,3 +113,5 @@ replace (
 
 	kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
 )
+
+go 1.13
```

Comment:
:( sorry, can you remove this. I know your IDE automatically adds it.

Reply:
This is added back by `make generate`.
Why we don't want this line in `go.mod`? Isn't is explicitly says that we now require go 1.13 to properly get all the dependencies and work?